hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1665,16 +1663,6 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, fmt.Errorf("failed to parse release image version: %w", err)
 	}
 
-	// In >= 4.11 We want to move most of the components reconciliation down to the CPO https://issues.redhat.com/browse/HOSTEDCP-375.
-	// For IBM existing clusters < 4.11 we need to stay consistent and keep deploying existing pods to satisfy validations.
-	// TODO (alberto): drop this after dropping < 4.11 support.
-	if !controlPlaneOperatorManagesMachineAutoscaler {
```

Comment:
We should double check whether this can be removed

Reply:
that's what I'm not sure about. It may still be needed for ROKS or satellite

Reply:
confirmed with Hide, they don't have any clusters < 4.11